src/components/bottom_bar/bottom_bar_sheet/bottom_bar_sheet.xml

```diff
@@ -3,6 +3,7 @@
     <Ripple>
       <div
         class="o-sheet d-flex align-items-center user-select-none text-nowrap "
+        tabindex="-1"
```

Comment:
why? ^^

Reply:
So it's focusable and it is the relatedTarget of the `blur` event. Otherwise the related target is just the document's body

Reply:
Well only on the elements that the composer cares about to decide if it should cancel the edition or not. So just this and the grid. On any ohter element, we dont really care and the composer stops the edition.